### PR TITLE
LIIKUNTA-522 | Refactor venue-graphql-proxy, add department data, enable Hauki support

### PR DESCRIPTION
**Pull request by karisal-anders**
### feat(venue-proxy): add department data, enable Hauki support, refactor

- add department field
- add organization field
- fix & enable Hauki support

refs LIIKUNTA-522

### Closes

**[LIIKUNTA-522](https://helsinkisolutionoffice.atlassian.net/browse/LIIKUNTA-522)**

## Additional notes

[LIIKUNTA-522]: https://helsinkisolutionoffice.atlassian.net/browse/LIIKUNTA-522?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by terovirtanen** [comment]
**Hobbies-Helsinki branch is deployed to platta: https://harrastukset-pr464.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**Events-Helsinki branch is deployed to platta: https://tapahtumat-pr464.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**Sports-Helsinki branch is deployed to platta: https://liikunta-pr464.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**VENUE GRAPHQL PROXY is deployed to platta: https://venue-graphql-proxy-pr464.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**EVENTS GRAPHQL PROXY is deployed to platta: https://events-graphql-proxy-pr464.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://tapahtumat-pr464.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://harrastukset-pr464.dev.hel.ninja :laughing::tada::tada::tada:

**Review by nikomakela — APPROVED** [review]
Nice! 👍

File: proxies/venue-graphql-proxy/src/types.ts [hunk 1]
```diff
@@ -34,13 +49,15 @@ type VenueDetails<T extends VenueDetailsTextType> = {
   name: T | null;
   infoUrl: T | null;
   streetAddress: T | null;
-  telephone: T | null;
-  ontologyTree: Ontology[];
-  ontologyWords: Ontology[];
+  telephone: string | null;
```
[the remark this answers is not in the document]
Comment (karisal-anders) [reply proxies/venue-graphql-proxy/src/types.ts:52]: The telephone type was unnecessarily a translatable string (i.e. TranslationsObject which had 3 locales in it) before even though it had only 1 locale to begin with. So yes, it has really changed as it was unnecessary to use a TranslationsObject for it before. Just cleared that up by making it use only 1 locale directly.

**Pull request merged**